tests/checkpatch_wrapper_test.sh

```diff
@@ -48,31 +31,31 @@ function checkpatch_helper
       ['check']=CHECK_MSG \
   )
 
-  res=$(execute_checkpatch "$TMP_TEST_DIR/samples/codestyle_$type_msg.c" 2>&1)
+  res=$(execute_checkpatch "$SHUNIT_TMPDIR/samples/codestyle_$type_msg.c" 2>&1)
```

Comment:
Let's document this temporary directory and write a paragraph or two explaining how and when to use it.